src/main/java/frc/util/SwerveModuleMath.java

```diff
@@ -0,0 +1,69 @@
+package frc.util;
+
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.kinematics.SwerveDriveKinematics;
+import edu.wpi.first.math.util.Units;
+
+public class SwerveModuleMath {
+    public static double restrictAngle(double angle) {
+        while (angle > 360) {
+            angle -= 360;
+        }
+        while (angle < 0) {
+            angle += 360;
+        }
+        return angle;
+    }
+
+    public static final SwerveDriveKinematics swerveMagik = new SwerveDriveKinematics(
+            new Translation2d(Units.inchesToMeters(24) / 2, Units.inchesToMeters(24) / 2), //FR
+            new Translation2d(Units.inchesToMeters(24) / 2, -Units.inchesToMeters(24) / 2), //FL
+            new Translation2d(-Units.inchesToMeters(24) / 2, Units.inchesToMeters(24) / 2), //BR
+            new Translation2d(-Units.inchesToMeters(24) / 2, -Units.inchesToMeters(24) / 2) //BL
```

Comment:
Make a variable for robotWidth and robotLength

Reply:
ignore the outdates all of these things still need to be fixed